cmd/stackit-api-manager/cmd/project.go

```diff
@@ -17,6 +17,7 @@ var (
 	stage                 string
 	openAPISpecFilePath   string
 	printJSON             bool
+	traceIDEnabled        bool
```

Comment:
keep it consistent with the flag above
```suggestion
	printTraceID        bool
```

Reply:
done